enzyme/Enzyme/GradientUtils.h

```diff
@@ -1263,24 +1273,27 @@ if (AtomicAdd) {
                          cast<PointerType>(ptr->getType())->getAddressSpace()));
     dif = BuilderM.CreateBitCast(dif, IntToFloatTy(dif->getType()));
   }
-  if (llvm::Triple(newFunc->getParent()->getTargetTriple()).getArch() ==
-          Triple::nvptx ||
-      llvm::Triple(newFunc->getParent()->getTargetTriple()).getArch() ==
-          Triple::nvptx64) {
-    if (dif->getType()->isFloatTy()) {
-      // auto atomicAdd =
-      cast<CallInst>(BuilderM.CreateCall(
-          Intrinsic::getDeclaration(newFunc->getParent(),
-                                    Intrinsic::nvvm_atomic_add_gen_f_sys,
-                                    {dif->getType(), ptr->getType()}),
-          {ptr, dif}));
-    } else if (dif->getType()->isDoubleTy()) {
+  if (false &&
```

Comment:
Probably want to delete this code before landing.

Reply:
Agreed, removed